bin/pgbackrest-repo/pgbackrest-repo.sh

```diff
@@ -67,10 +81,8 @@ then
 	
 fi
 
-mkdir -p ~/.ssh/
-cp $CONFIG/config ~/.ssh/
 cp $CONFIG/id_ed25519 /tmp
-chmod 400 /tmp/id_ed25519 ~/.ssh/config
```

Comment:
This _might_ let us clear one of the emptyDirs, so noting that.

Reply:
I did remove the `pgbackrest-ssh` volume, which was mounted to `/var/lib/pgsql/.ssh`.